### DISCUSSION TITLE
Is there a way to delete disabled repository?

### DISCUSSION THREAD

**Discussion by rairulyle** [community/community#27852] [Other Feature Feedback, Questions, & Ideas]
I have this repository that has been disabled by Github and I wanted to delete it. Is there an easy way to do this?

![image](https://user-images.githubusercontent.com/12708526/182078683-aaef1d7e-079c-456b-aa25-af94015d5525.png)

**Comment by jfullstackdev** [comment]
is that link pointing to this one 

https://support.github.com/request

but I'm not very sure what category is that because I don't know why it's disabled in the first place
